README.md

```diff
@@ -128,8 +128,11 @@ Please install the ```tree``` command before moving forward:
 
    - MacOSX
         ```
-        install tree - http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
-        install wget - http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
+        brew install tree 
```

Comment:
Would you mind preserving the original steps and adding a sub-topic where you add your instructions?

So it's clear that if you using a certain version to install using brew and the alternative would be to go to the respective links